src/include/liburing.h

```diff
@@ -152,7 +156,7 @@ struct io_uring_probe *io_uring_get_probe(void);
  */
 void io_uring_free_probe(struct io_uring_probe *probe);
 
-static inline int io_uring_opcode_supported(const struct io_uring_probe *p,
+LIB_URING_STATIC_INLINE int io_uring_opcode_supported(const struct io_uring_probe *p,
```

Comment:
The very first one is a good example - first line now is too long, and the 2nd line is just lazily hanging now with deprecated spacing.